include/api.php

```diff
@@ -2918,7 +2919,7 @@ function api_direct_messages_box($type, $box, $verbose) {
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.